src/extendedstorage.cpp

```diff
@@ -142,25 +158,27 @@ void ExtendedStorage::clearLoaded()
 }
 
 bool ExtendedStorage::getLoadDates(const QDate &start, const QDate &end,
-                                   QDateTime &loadStart, QDateTime &loadEnd)
+                                   QDateTime &loadStart, QDateTime &loadEnd) const
```

Comment:
Old but could help code readability if the latter two out parameters were pointers.